modules/using-assisted-installer-oci-agent-iso.adoc

```diff
@@ -6,62 +6,80 @@
 [id="using-assisted-installer-oci-agent-iso_{context}"]
 = Using the {ai-full} to generate an {oci}-compatible discovery ISO image
 
-Generate a discovery ISO image and upload the image to {oci-first}, so that the agent can perform hardware and network validation checks before you install an {product-title} cluster on {oci}.
+Create the cluster configuration and generate the discovery ISO image in the {ai-full} web console. 
 
-From the {oci} web console, you must create the following resources:
+.Prerequisites
 
-* A compartment for better organizing, restricting access, and setting usage limits to {oci} resources.
-* An object storage bucket for safely and securely storing the discovery ISO image. You can access the image at a later stage for the purposes of booting the instances, so that you can then create your cluster.
+* You created a child compartment and an object storage bucket on {oci}. For details, see _Preparing the {oci} environment_. 
+* You reviewed details about the {product-title} installation and update processes. 
 
-.Prerequisites
+== Creating the cluster
 
-* You created a child compartment and an object storage bucket on {oci}. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-assisted.htm#provisioning-cloud-infrastructure[Provisioning Cloud Infrastructure ({oci} Console)] in the Oracle documentation.
-* You reviewed details about the {product-title} installation and update processes.
-* If you use a firewall and you plan to use a Telemetry service, you configured your firewall to allow {product-title} to access the sites required.
-* Before you create a virtual machines (VM), see link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types (Red Hat Ecosystem Catalog portal)] to identify the supported {oci} VM shapes.
+Set the cluster details. 
 
 .Procedure
 
-. From the link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[*Install OpenShift with the Assisted Installer*] page on the Hybrid Cloud Console, generate the discovery ISO image by completing all the required {ai-full} steps.
-+
-.. In the *Cluster Details* step, complete the following fields:
+. Log into link:https://console.redhat.com/[{ai-full} web console] with your credentials.
+
+. In the *Red Hat OpenShift* tile, select *OpenShift*. 
```

Comment:
I assume all these uses of "OpenShift" are how they appear in the GUI. As a team, we probably need to revisit adding an attribute for such cases.